### ISSUE TITLE
PanoViewer type panorama cannot view full width of image

### ISSUE THREAD

**Issue by phanpiseth9** [naver/egjs-view360#290]
## Description

PanoViewer type panorama cannot view full width of image,please help.
testing in https://naver.github.io/egjs-view360/examples/panoviewer/projection-type/panorama_image.html
https://www.dropbox.com/s/w6wqyree9pxmjdv/photo_2019-09-18_17-55-44.jpg?dl=0

**Comment by jongmoon** [comment]
@pahnpiseth Thank you for reporting. I'll check soon why error occurs.

**Comment by jongmoon** [comment]
@pahnpiseth Could tell me the environment where error occurs?

I tested it with latest(3.2.2-rc) version and I tested it on following environment. and It works well.

Test Code

```html
<div class="photo360-container">
        <div class="photo360" id="myPanoViewer"></div>
</div>
```

```js
var container = document.getElementById("myPanoViewer");
var panoViewer = new PanoViewer(container, {
            image: "../img/PanoViewer/Panorama/panorama-picture.jpg", /* your picture */
            projectionType: "panorama"
        }
);
```


  - Test 1
    - OS: Mac OS Majave (10.14.5)
    - Browser: Chrome(76) , FF(65)

  - Test2
    - OS: iOS 12.2
    - Browser: Safari

![image](https://user-images.githubusercontent.com/3255471/65141524-c5f3c380-da4b-11e9-8ae9-ef41c87ff931.png)

**Comment by jongmoon** [comment]
@pahnpiseth Would you tell me the what is the expected result?

**Comment by pahnpiseth** [comment]
![IMG_20190825_152732-min](https://user-images.githubusercontent.com/13740546/65150763-f38e3c00-da4e-11e9-8ea1-a48f90282b24.jpg)

I cannot scroll to last

**Comment by pahnpiseth** [comment]
![Screenshot_20190918-195827__01](https://user-images.githubusercontent.com/13740546/65150917-3d772200-da4f-11e9-81a0-448308f0e43d.jpg)
I cannot scroll to last.

**Comment by pahnpiseth** [comment]
Please help!!

**Comment by jongmoon** [comment]
@pahnpiseth OK, I'll check it!

**Comment by pahnpiseth** [comment]
Thanks you so much.

**Comment by jongmoon** [comment]
@pahnpiseth I could fix it as follows. But it needs to be tested. After tested I would apply on next release.

![image](https://user-images.githubusercontent.com/3255471/65232243-c94e8400-db0b-11e9-9ba3-5f020fc9a109.png)

**Comment by pahnpiseth** [comment]
Thanks you so much brother.

**Comment by jongmoon** [comment]
@pahnpiseth I fixed it and view360 is published as 3.2.2 version.
